### PR TITLE
Adding tagged images for all 7.x versions of PHP

### PR DESCRIPTION
**Pull request by pbarabe**
Implements build files for 7.0, 7.1, 7.2, and 7.3

### CODE REVIEW

**Comment by pbarabe** [comment]
@uacate Please review & complete the PR when you have a moment. Thx!

File: 7.0/Dockerfile [hunk 1]
```diff
@@ -32,10 +39,12 @@ RUN apt-get update && apt-get install -y \
   php-memcached \
   php-mysql \
   php-soap \
-  php-xml \
-  unzip \
-  zip
+  php-xml
```
Comment (uacate) [inline 7.0/Dockerfile]: When I have used this same Dockerfile for other projects, sometimes I have gotten feedback from the docker instance that php-mbstring.

Comment (uacate) [reply 7.0/Dockerfile]: Nevermind.  I see this in the code below.

**Review by uacate — COMMENTED** [review]
I cloned my fork locally and ran the docker build command provided in the README and it worked great.

I see php-mbstring in some versions of the Dockerfile, not in the 7.0 version.  In some cases of using Docker and php for gradapp and other projects.  The php code has barked about not having the php-mbstring dep.  I suspect this maybe me not understanding current versions of php not needing mbstring.

**Comment by pbarabe** [comment]
@uacate The mbstring package has been abandoned and so there is no php-mbstring package for PHP 7.2+. The recommended alternative is php-sodium, which is included in the 7.2 & 7.3 builds. I'm trying to find the resource where I found that out yesterday and will add it to the discussion.

php-mbstring is still available for 7.0 & 7.1 and is include in those images (if you're looking in the PR diff, it's in the collapsed section of the diff).

**Review by uacate — APPROVED** [review]
Looks good.  Ran docker build on local forked and worked as described.

**Pull request merged**